components/org.wso2.identity.event.common.publisher/src/main/java/org/wso2/identity/event/common/publisher/model/common/SimpleSubject.java

```diff
@@ -122,4 +126,14 @@ public static SimpleSubject createURISubject(String uri) {
         return subject;
     }
 
+    public static SimpleSubject createAliasesSubject(List<SimpleSubject> identifiers) {
+
+        if (identifiers == null || identifiers.isEmpty()) {
+            return null;
+        }
+        SimpleSubject subject = new SimpleSubject();
```

Comment:
Shall we check the validations as discussed offline ? 

1. Value SHOULD NOT contain exact duplicates.
2. Subject Identifiers MUST NOT be nested
3. MUST NOT contain a Subject Identifier in the Aliases Identifier Format.